doc/nodejs.rst

```diff
@@ -0,0 +1,57 @@
+Usage with Node.js
+==================
+
+Although remoteStorage.js was initially written for being used in a
+browser, we do support using it in a Node.js environment as well.
+
+Installation
+------------
+
+Client library
+^^^^^^^^^^^^^^
+
+remoteStorage.js is available from npm::
+
+   $ npm install remotestoragejs
+
+Modules
+^^^^^^^
+
+remoteStorage.js modules are not packaged as part of the ``remotestoragejs``
+npm module. You will need to download (or create) your own module to use. You
+can visit our `modules repository`_ to see existing modules, feel free to
+submit a pull request if you've written your own.
+
+They can also be published as npm modules: `search for "remotestorage-module" on npm`_ to find them.
+
+As of remoteStorage.js 1.0 you do not need to declare a ``remoteStorage`` global
+for modules to work in Node.js any longer, because they can be ES6 modules.
```

Comment:
1. Modules need to be documented for everyone, hence #1004 
2. I think information about no longer having to use a global is irrelevant for this document. If anything, it belongs in the CHANGELOG or announcement post imo.